README.md

```diff
@@ -20,6 +20,28 @@ This action for [Changesets](https://github.com/changesets/changesets) creates a
 - published - A boolean value to indicate whether a publishing has happened or not
 - publishedPackages - A JSON array to present the published packages. The format is `[{"name": "@xx/xx", "version": "1.2.0"}, {"name": "@xx/xy", "version": "0.8.9"}]`
 
+### Permissions
```

Comment:
It might be worth mentioning that when publishing with provenance, you'll need `id-token: write` too. An example of how this can be configured: https://github.com/changesets/changesets/pull/1659/files 

Note there are other ways to enable this - but they all require this permission.

Reply:
I can add that in as an optional permission